polling-publisher/README.md

```diff
@@ -0,0 +1,47 @@
+# Polling Publisher-Subscriber Microservice Pattern
```

Comment:
README.md has a specific format. See https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute